ide/projectuiapi.base/src/org/netbeans/api/project/ui/OpenProjects.java

```diff
@@ -315,5 +314,24 @@ public void addProjectGroupChangeListener( @NonNull ProjectGroupChangeListener l
     public void removeProjectGroupChangeListener( @NonNull ProjectGroupChangeListener listener) {
         trampoline.removeProjectGroupChangeListenerAPI(listener);
     }
-    
+
+    /** Creates new {@link ExplorerManager} to show and control
+     * logical structure of a project.
+     *
+     * @return new instance of the manager
+     * @since 1.100
+     */
+    public ExplorerManager createLogicalView() {
```

Comment:
This API has to be documented in apichanges. Sorry for not doing that to begin with.